cmd/ocm-backplane/cloud/credentials_test.go

```diff
@@ -145,67 +141,6 @@ var _ = Describe("Cloud console command", func() {
 		mockCtrl.Finish()
 	})
 
-	Context("test get Cloud Credential", func() {
-
-		It("should return AWS cloud credential", func() {
-
-			mockClientUtil.EXPECT().GetBackplaneClient(proxyURI).Return(mockClient, nil).AnyTimes()
-			mockClient.EXPECT().GetCloudCredentials(gomock.Any(), trueClusterID).Return(fakeAWSResp, nil)
-
-			crdentialResponse, err := getCloudCredential(proxyURI, trueClusterID)
-			Expect(err).To(BeNil())
-
-			Expect(crdentialResponse.JSON200).NotTo(BeNil())
-
-		})
-
-		It("should fail when AWS Unavailable", func() {
-			fakeAWSResp.StatusCode = http.StatusInternalServerError
-
-			mockClientUtil.EXPECT().GetBackplaneClient(proxyURI).Return(mockClient, nil).AnyTimes()
-			mockClient.EXPECT().GetCloudCredentials(gomock.Any(), trueClusterID).Return(fakeAWSResp, nil)
-			_, err := getCloudCredential(proxyURI, trueClusterID)
-			Expect(err).NotTo(BeNil())
-
-			Expect(err.Error()).To(ContainSubstring("error from backplane: \n Status Code: 500\n"))
-
-		})
-
-		It("should fail when GCP Unavailable", func() {
-			fakeGCloudResp.StatusCode = http.StatusInternalServerError
-
-			mockClientUtil.EXPECT().GetBackplaneClient(proxyURI).Return(mockClient, nil).AnyTimes()
-			mockClient.EXPECT().GetCloudCredentials(gomock.Any(), trueClusterID).Return(fakeGCloudResp, nil)
-			_, err := getCloudCredential(proxyURI, trueClusterID)
-			Expect(err).NotTo(BeNil())
-
-			Expect(err.Error()).To(ContainSubstring("error from backplane: \n Status Code: 500\n"))
-
```

Comment:
Any reson to remove existing tests with AWS v2 approch ?

Reply:
I'm ok with receiving pushback on this, but to share my reasoning mostly because they aren't passing anymore and it's critical to make sure that osdctl AWS calls are going through the proxy haha.

My personal preference is to refactor/add some tests back in a future PR, but I can add it into this PR if that's best. My functional "changes" are in the first commit in the PR and you can see all the refactored/mostly removed tests in the second commit.

Reply:
Good point, @mjlshen .. Let's add/refactor unit-test in future PR's .. ( we are on a journey to reach 80% code coverage for bp-cli  :) )